src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -28,6 +28,8 @@ object TaskGroupBuilder {
     ).asJava)
     .build
 
+  val ephemeralVolPathPrefix = "volumes/"
```

Comment:
/cc @sascala 

ephemeral volumes are hosted in the executor sandbox at `volume/{volume-name}` and are "linked" to by sub-containers. "linked" meaning that if `filesystem/posix` isolation is used by the Mesos agent then it's symlinked to the `volumeMount.path` of the sub-container; if `filesystem/linux` isolation is being used instead (default for dc/os) then bind mounting is used.